hil-test/tests/spi_full_duplex.rs

```diff
@@ -1,7 +1,7 @@
 //! SPI Full Duplex test suite.
 
 //% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s2 esp32s3
-//% FEATURES: generic-queue
+//% FEATURES: generic-queue defmt
```

Comment:
I don't think we want to keep `defmt`

Reply:
dammit I always forget to remove these

Reply:
Protip, add it as a default feature in cargo.toml (and don't commit it).

Reply:
As you can see, my issue is with the "don't commit it" part :)

Reply:
I was thinking about a pre-commit-hook which checks for a keyword like STOPSHIP or something - committing stuff I don't want to commit happens too often for me, too